packages/auth/test/integration/webdriver/redirect.test.ts

```diff
@@ -39,8 +39,13 @@ import { START_FUNCTION } from './util/auth_driver';
 use(chaiAsPromised);
 
 browserDescribe('WebDriver redirect IdP test', driver => {
```

Comment:
Can we remove the version pinning in this PR as well, or will you be sending a follow up?

Reply:
Pinned to version 114 as I still see the `"No such object: chromedriver/LATEST_RELEASE_115.0.5790"` failure. Please see the PR description for more details.

Added a TODO to investigate and address this install failure.